### PR TITLE
Opinions test

### PR DESCRIPTION
**Pull request by yhordii**

### CODE REVIEW

File: src/test/java/com/qaprosoft/carina/demo/MyWebTest.java [hunk 1]
```diff
@@ -217,6 +217,32 @@ public void phoneFinderVerifying() {
         Assert.assertTrue(phoneFinderPage.isCurrectBrandPresented(R.TESTDATA.get("finder_phone_brand")), "Searched brand isn't presented.");
         Assert.assertTrue(phoneFinderPage.isNoteOnResultPagePresented(), "Note text isn't presented on the result page.");
         Assert.assertTrue(phoneFinderPage.returnToPhoneFinder(), "Can't return to previous menu.");
+        getDriver().close();
```
Comment (yhordii) [inline src/test/java/com/qaprosoft/carina/demo/MyWebTest.java]: tested out, forgot to remove

Comment (vrudyk) [reply src/test/java/com/qaprosoft/carina/demo/MyWebTest.java]: remove now

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/PhoneModelPage.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        opinionsSort.select("Newest first");
+    }
+
+    public boolean areCommentsSortedByDate() {
+        List<Date> dates = new ArrayList<>();
+        List<Integer> hours = new ArrayList<>();
+        dateComments.stream().forEach(t -> {
+            if (t.getText().contains("ago")) {
+                hours.add(Integer.parseInt(t.getText().replaceAll("\\D+", "")));
+            } else {
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/gui/pages/PhoneModelPage.java]: you forgot minutes, separate this
add logic for 'minutes' and 'hours' separately

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/PhoneModelPage.java [hunk 3]
```diff
@@ -90,8 +91,14 @@ public void sortByNewFirst() {
...
         List<Integer> hours = new ArrayList<>();
+        List<Integer> minutes = new ArrayList<>();
+        List<Integer> seconds = new ArrayList<>();
         dateComments.stream().forEach(t -> {
-            if (t.getText().contains("ago")) {
+            if (t.getText().contains("seconds ago") || t.getText().contains("second ago")) {
+                seconds.add(Integer.parseInt(t.getText().replaceAll("\\D+", "")));
+            } else if (t.getText().contains("minutes ago") || t.getText().contains("minute ago")) {
+                minutes.add(Integer.parseInt(t.getText().replaceAll("\\D+", "")));
+            } else if (t.getText().contains("hours ago") || t.getText().contains("hour ago")) {
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/gui/pages/PhoneModelPage.java:101]: if you use contains remove condition with letter s
or replace all contains to equals

**Comment by vrudyk** [comment]
Ok

**Pull request merged**